autoware_launch/launch/autoware.launch.xml

```diff
@@ -38,6 +38,8 @@
   <!-- Perception -->
   <arg name="perception_mode" default="lidar" description="select perception mode. camera_lidar_radar_fusion, camera_lidar_fusion, lidar_radar_fusion, lidar, radar"/>
   <arg name="traffic_light_recognition/enable_fine_detection" default="true" description="enable traffic light fine detection"/>
+  <!-- Localization -->
+  <arg name="localization_mode" default="ndt" description="select localization mode. ndt, yabloc"/>
```

Comment:
I would like to make modes like lidar or camera if possible.

Reply:
I changed mode names in 6d91776. Now the users can launch as `localization_mode:=lidar` or `localization_mode:=camera` 
Is this what you wish?

Reply:
Yes 👍